File: Selenium/Project/FST_HRM_Project/src/test/java/Activity/Activity_05.java
package Activity;
//import org.testng.Assert;
import dev.failsafe.internal.util.Assert;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

import java.time.Duration;

public class Activity_05 {
    public static void main(String[] args) throws InterruptedException {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("http://alchemy.hguy.co/orangehrm");
        //LOGIN
        driver.findElement(By.xpath("//input[@name='txtUsername']")).sendKeys("orange");
        driver.findElement(By.xpath("//input[@id='txtPassword']")).sendKeys("orangepassword123");
        driver.findElement(By.id("btnLogin")).click();
        Thread.sleep(3000);
        driver.findElement(By.id("menu_pim_viewMyDetails")).click();
        Thread.sleep(3000);
        //Edit steps
        driver.findElement(By.id("btnSave")).click();
        driver.findElement(By.id("personal_txtEmpFirstName")).clear();
        driver.findElement(By.id("personal_txtEmpLastName")).clear();
        driver.findElement(By.id("personal_txtEmpFirstName")).sendKeys("Iksha");
        driver.findElement(By.id("personal_txtEmpLastName")).sendKeys("tg");
        Thread.sleep(3000);
        driver.findElement(By.id("personal_optGender_2")).click();
        driver.findElement(By.id("personal_DOB")).clear();
        Thread.sleep(3000);
        driver.findElement(By.id("personal_DOB")).sendKeys("2022-09-08");
        driver.findElement(By.id("btnSave")).click();
        Thread.sleep(3000);
        driver.close();

        //select.selectByValue("Indian");


    }
}

File: Python/Activities/Project/Activity_05.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.support.wait import WebDriverWait

service = FirefoxService(GeckoDriverManager().install())

# start the driver
with webdriver.Firefox(service=service) as driver:
    # Navigate to the URL
    driver.get("http://alchemy.hguy.co/orangehrm")
    actualTitle = driver.title
    print("The Title of the Page is: ", actualTitle)
    driver.find_element(By.XPATH,"//input[@name='txtUsername']").send_keys("orange")
    driver.find_element(By.XPATH,"//input[@id='txtPassword']").send_keys("orangepassword123")
    driver.find_element(By.ID,"btnLogin").click()
    driver.find_element(By.ID,"menu_pim_viewMyDetails").click()
    driver.find_element(By.ID,"btnSave").click()
    driver.find_element(By.ID,"personal_txtEmpFirstName").clear()
    driver.find_element(By.ID,"personal_txtEmpFirstName").send_keys("ABCD")
    driver.find_element(By.ID,"personal_txtEmpLastName").clear()
    driver.find_element(By.ID,"personal_txtEmpLastName").send_keys("ABCD")
    driver.find_element(By.ID,"personal_optGender_2").click()
    driver.find_element(By.ID,"personal_DOB").clear()
    driver.find_element(By.ID,"personal_DOB").send_keys("1990-01-21")
   # driver.find_element(By.ID,"btnSave").click()
    driver.close()







